protocols/example/crud/shoppingcart/persistence/domain.proto

```diff
@@ -1,19 +1,43 @@
+// Copyright 2019 Lightbend Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 // These are the messages that get persisted - the events, plus the current state (Cart) for snapshots.
-syntax = "proto3";
+    syntax = "proto3";
```

Comment:
regarding line  20:
the go package cannot have dots, a package should be fully qualified i,port path or better, a local package name like "persistence".

A composed name like "shoppingcart/persistence" we used for now in other protos is actually not allowed, and it will be fixed with the upcoming 0.2.0 release.

Reply:
understood